test/test_k3portlock.py

```diff
@@ -128,3 +130,21 @@ def test_sleep_time(self):
             self.assertTrue(0.9 < t1 - t0 < 1.1,
                             'sleep_time should not affect timeout(1 sec),'
                             ' but it spends {0} seconds'.format(t1 - t0))
+
+    def test_collision(self):
+        resource.setrlimit(resource.RLIMIT_NOFILE, (102400, 102400))
+
+        dd = {}
+        ls = []
+        for i in range(1 << 15):
+            key = str(hashlib.sha1(str(i).encode("utf8")).hexdigest())
+            lck = key
+            print('lock is', i, lck)
+            l = k3portlock.Portlock(lck, timeout=8)
+            r = l.try_lock()
+            if not r:
+                print('collide', i, l.addr)
+                print(l.socks)
+
+            dd[l.addr] = i
+            ls.append(l)
```

Comment:
Hm...I mean, make it a real **test**, with assertion that the collision rate should be slower than some value etc.